content/docs/getting-started.md

```diff
@@ -18,19 +18,19 @@ redirect_from:
   - "docs/environments.html"
 ---
 
-This page is an overview of the React documentation and related resources.
+यह पन्ना रियेक्ट के प्रलेखन और उससे सम्बंधित साधनो  का अवलोकन है | 
```

Comment:
We are not translating words like React, Overview. You can get complete list of words which we are not translating [here](https://github.com/reactjs/hi.reactjs.org/issues/2)